src/bacnet/basic/object/nc.c

```diff
@@ -433,6 +433,24 @@ bool Notification_Class_Write_Property(BACNET_WRITE_PROPERTY_DATA *wp_data)
                 uint32_t device_id;
                 BACNET_DESTINATION *destination;
                 BACNET_RECIPIENT *recipient;
+                uint8_t *ft_hour = &TmpNotify.Recipient_List[idx].FromTime.hour;
```

Comment:
you probably could have  **uint8_t** ft_hour = &TmpNotify.Recipient_List[idx].FromTime.hour; here and avoid pointer dereferencing below.... not a bi deal...

Reply:
I agree I dont think there is any point of having pointers here if we're just manipulating the value